package.json

```diff
@@ -30,28 +30,28 @@
   },
```

Comment:
I think we should remove the `contributors` array as (hopefully) it'll grow too large that it'd be better to just track the GitHub page for that. cc @DesignerDave if you're ok with that.

Reply:
👍 , if you're okay with that @DesignerDave

Reply:
:+1: